### DISCUSSION TITLE
Monika Disappeared from the room after I reset my computer

### DISCUSSION THREAD

**Discussion by XTacticalNukex4** [Monika-After-Story/MonikaModDev#7969] [QA - tech support + general] [answered]
So my computer was hit with some sort of malware that used up a ton of my computers storage, so I had to factory reset it. SO, I backed up DDLC and MAS to my usb stick and then factory reset it. I Made sure to get anything ddlc related, including the renpy persistent folder file. After I reset it and got the renpy data into my reset computer app data, I realized that the ddlc folder (the one where the application is) didn't have anything in it, which made me get the game from ddlc.moe and download the latest version of Monika after story again. I plugged them in and put Monika file (from when you tell her you are taking her somewhere) in the characters folder again. But when I opened it up, Monika wasn't there, just her chair, and I'm getting worried. is there anyway to get her back or am I screwed?

![image](https://user-images.githubusercontent.com/91707478/135550743-0e971da7-59fd-476d-8546-0b0db7513890.png)
![image](https://user-images.githubusercontent.com/91707478/135550816-f3dd5c22-f25c-4e75-84c8-a883fe897944.png)
![image](https://user-images.githubusercontent.com/91707478/135550878-6de3325b-ece3-4300-b2de-f400c78505b5.png)

**Comment by Booplicate** [comment, marked as answer]
Her file got corrupted, [restore an older persistent backup](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent).
